### PR TITLE
[Components with more than 1 service] LPS-183543 Refactor ProductNavigationControlMenuEntry components from analytics, layout and segments modules 

### PR DESCRIPTION
**Pull request by RenanUmV**
[LPS-183543](https://issues.liferay.com/browse/LPS-183545)

### CODE REVIEW

**Comment by RenanUmV** [comment]
ci:test:help

**Comment by RenanUmV** [comment]
ci:test:echo

**Comment by RenanUmV** [comment]
ci:test:analytics-cloud

**Comment by catarineaguiar** [comment]
ci:test:analytics-cloud-acceptance

File: modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/product/navigation/control/menu/AnalyticsReportsProductNavigationControlMenuEntry.java [hunk 1]
```diff
@@ -167,17 +165,6 @@ public boolean includeIcon(
 		return true;
 	}
 
-	public boolean isPanelStateOpen(HttpServletRequest httpServletRequest) {
```
Comment (rafaprax) [inline modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/product/navigation/control/menu/AnalyticsReportsProductNavigationControlMenuEntry.java:170]: Hey, @RenanUmV move these methods to the interface and add the default implement of them into the `BaseProductNavigationControlMenuEntry` class. Thx

**Review by rafaprax — CHANGES_REQUESTED** [review]

File: modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/product/navigation/control/menu/AnalyticsReportsProductNavigationControlMenuEntry.java (hunk 1 above)
Comment (rafaprax) [reply modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/product/navigation/control/menu/AnalyticsReportsProductNavigationControlMenuEntry.java:170]: Apply this for the others methods you have extract as well

**Pull request closed without merging**